docs/configuration.rst

```diff
@@ -221,6 +221,40 @@ Panel options
   The SQL panel highlights queries that took more that this amount of time,
   in milliseconds, to execute.
 
+* ``PRETTIFY_SQL``
```

Comment:
Can you move this above `PROFILER_MAX_DEPTH` to make the settings alphabetical please?

Reply:
Done in [cbf8f84](https://github.com/jazzband/django-debug-toolbar/pull/1438/commits/cbf8f84c06264060033780a70e1155e2503c88c5).